package.json

```diff
@@ -34,15 +34,14 @@
     "postpack": "ember ts:clean"
   },
   "dependencies": {
-    "@ember/render-modifiers": "^2.0.0",
     "@tailwindcss/typography": "^0.5.2",
     "ember-auto-import": "^2.2.0",
     "ember-cli-babel": "^7.26.3",
     "ember-cli-htmlbars": "^6.0.1",
     "ember-cli-typescript": "^5.1.0",
     "ember-concurrency": "^2.1.2",
     "ember-element-helper": "^0.6.0",
-    "ember-modifier": "^3.0.0",
+    "ember-modifier": "4.0.0-beta.1",
```

Comment:
Not that this is a problem, but just calling it out,
this _may_ be a breaking change for consumers. _but_ since we are still pre v1, it's waay less of an issue